examples/tests/quotes.earth

```diff
@@ -17,9 +18,18 @@ test-env-quoting:
     RUN echo "b25ea0182bf88bf6c47daaf2e4da5cac  -" > /expected.md5
     RUN diff /actual.md5 /expected.md5
 
-test-env-quoting2:
-    ENV BACKSLASH "\\"
-    RUN test "$BACKSLASH" = "\\"
+test-env-unquoted-backslash:
+    RUN touch /before-backslash
+    RUN echo \\
+    RUN touch /after-backslash
+    RUN ls /before-backslash
+    RUN ls /after-backslash
```

Comment:
this right here is failing; since the `RUN echo \\` line is turning into `RUN echo \    RUN touch /after-backslash`